homeassistant/components/zha/fan.py

```diff
@@ -142,15 +132,32 @@ async def async_added_to_hass(self):
         )
 
     @property
-    def speed(self) -> Optional[str]:
-        """Return the current speed."""
-        return VALUE_TO_SPEED.get(self._fan_channel.fan_mode)
+    def percentage(self) -> str:
```

Comment:
```py
def percentage(self) -> Optional[int]:
```

Reply:
Fixed in https://github.com/home-assistant/core/pull/46714